scripts/scil_split_volume_by_labels.py

```diff
@@ -15,6 +15,7 @@
 import os
 
 import nibabel as nib
+import numpy as np
```

Comment:
Is this used?

Reply:
@frheault quick fix and I merge + release.